autoload/vifm/colorconv.vim

```diff
@@ -1,13 +1,13 @@
 " Vim script that converts Vim colorschemes to Vifm
-" Author:      Roman Plšl
+" Author:      Roman Plášil
 " Maintainer:  xaizek <xaizek@posteo.net>
 " Last Change: September 21, 2021
 
 function! s:ConvertGroup(gr, to, deffg, defbg)
 	let syn = synIDtrans(hlID(a:gr))
-	let fg = synIDattr(syn, "fg")
-	let bg = synIDattr(syn, "bg")
-	let prefix = 'cterm'
+	let fg = synIDattr(syn, "fg#")
+	let bg = synIDattr(syn, "bg#")
```

Comment:
I get the prefix change is because `deffg`/`defbg` aren't GUI colors and this is to always get `#rrbbgg` form and not color name, right?